src/test/java/subway/line/LineRestAssured.java

```diff
@@ -4,31 +4,61 @@
 import io.restassured.response.ExtractableResponse;
 import io.restassured.response.Response;
 import org.springframework.http.MediaType;
+import subway.line.presentation.LinePatchRequest;
 import subway.line.presentation.LineRequest;
 
 public class LineRestAssured {
 
     private static final String API_PATH = "/lines";
 
-    public static ExtractableResponse<Response> createLine(LineRequest lineRequest) {
+    private static final String APPLICATION_JSON_VALUE = MediaType.APPLICATION_JSON_VALUE;;
```

Comment:
```suggestion
    private static final String APPLICATION_JSON_VALUE = MediaType.APPLICATION_JSON_VALUE;
```

이건 굳이 이렇게 하지 않고 static import 해도 될 것 같아요 😅